parsers/htjson/htjson.go

```diff
@@ -203,3 +206,11 @@ func (p *Parser) tryTimeFormats(t string) time.Time {
 	}
 	return ts
 }
+
+func (p *Parser) warnAboutTime(fieldName string, foundTimeVal interface{}, msg string) {
+	if p.warnedAboutTime {
+		return
+	}
+	logrus.WithField("time_field", fieldName).WithField("time_value", foundTimeVal).Warn(msg + "\n  Please refer to https://honeycomb.io/docs/json#timestamp-parsing")
```

Comment:
<3 I was about to say "let's put the time field spec and contents in the error message" up above.  \o/ structured logging!